tests/python/relay/test_pass_partition_graph.py

```diff
@@ -903,17 +889,17 @@ def test_detect_pattern(pattern_table, include_bn, include_sigmoid,
 
     def test_partition():
         # conv + bn + relu, conv + relu -> fused conv_bias_relu, conv, and relu
-        test_detect_pattern([conv2d_bias_relu_pat], True, False, 3)
+        test_detect_pattern([dnnl_patterns[0]], True, False, 3)
```

Comment:
better to make variables `conv2d_bias_relu_pat` and `conv2d_relu_pat` so that
* make the diff smaller
* less change if later somebody adds new patterns to the table and `dnnl_patterns[0]` changes.